core/src/main/java/org/frankframework/jta/narayana/NarayanaConfigurationBean.java

```diff
@@ -23,7 +23,9 @@
 import javax.sql.DataSource;
 
 import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.Logger;
+import org.frankframework.core.JndiContextPrefixFactory;
+import org.frankframework.jndi.ObjectLocator;
+import org.frankframework.util.AppConstants;
```

Comment:
Any idea why the imports are all over the place?

Reply:
You mean, you don't like the way we've configured it in `.editorconfig`:
`ij_java_imports_layout = $*,|,java.**,|,javax.**,|,org.**,|,com.**,|,*`
I think a bit out of scope of my PR.

Reply:
We see the reorg of imports many times now, after we've renamed our packages. So it's coming up in many PRs...

I do think we should change that import spec though to make our own packages stand out. Out of the scope of your PR indeed but something that one of us should pick up soon in a separate PR.

Reply:
Let's first then refine the way it should be. That issue also does not describe the import issues you're trying to address here, even though the import change is totally valid.